### PR TITLE
Alpine65

### PR DESCRIPTION
**Pull request by Gondolindrim**
Intruduces Alpine65 support

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/alpine65/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2012,2013 Jun Wako <wakojun@gmail.com>
```
Comment (guilhermedeandrade) [inline keyboards/alpine65/keymaps/via/keymap.c:2]: I guess you forgot to update de copyright

File: keyboards/alpine65/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2012,2013 Jun Wako <wakojun@gmail.com>
```
Comment (guilhermedeandrade) [inline keyboards/alpine65/keymaps/default/keymap.c:2]: I guess you forgot to update the copyright

File: keyboards/alpine65/keymaps/via/keymap.c (hunk 1 above)
Comment (Gondolindrim) [reply keyboards/alpine65/keymaps/via/keymap.c]: Comitted in b2facc2

File: keyboards/alpine65/keymaps/default/keymap.c (hunk 2 above)
Comment (Gondolindrim) [reply keyboards/alpine65/keymaps/default/keymap.c]: Committed in 3240ab0

**Review by drashna — APPROVED** [review]

**Comment by Gondolindrim** [comment]
Rebased master as per zvecr's recommendation

**Review by ridingqwerty — APPROVED** [review]
Looks good to me.

**Comment by ridingqwerty** [comment]
Thanks!

**Pull request merged**